### PR TITLE
modify floating formulas

### PR DESCRIPTION
**Pull request by nuevoalex**
We want the decrease to happen at a slower rate than increase

### CODE REVIEW

**Review by adrake33 — APPROVED** [review]
LGTM

File: packages/augur-core/source/contracts/reporting/Universe.sol [hunk 1]
```diff
@@ -374,16 +374,16 @@ contract Universe is ITyped, IUniverse {
             _previousValue = _defaultValue;
         }
 
-        // Modify the amount based on the previous amount and the number of markets fitting the failure criteria. We want the amount to be somewhere in the range of 0.5 to 2 times its previous value where ALL markets with the condition results in 2x and 0 results in 0.5x.
+        // Modify the amount based on the previous amount and the number of markets fitting the failure criteria. We want the amount to be somewhere in the range of 0.9 to 2 times its previous value where ALL markets with the condition results in 2x and 0 results in 0.9x.
         // Safe math div is redundant so we avoid here as we're at the stack limit.
         if (_badMarkets <= _totalMarkets / _targetDivisor) {
-            // FXP formula: previous_amount * actual_percent / (2 * target_percent) + 0.5;
+            // FXP formula: previous_amount * actual_percent / (10 * target_percent) + 0.9;
```
Comment (epheph) [inline packages/augur-core/source/contracts/reporting/Universe.sol:380]: Pedantic, but this formula in comment isn't written correctly based on order-of-operations.

Comment (epheph) [reply packages/augur-core/source/contracts/reporting/Universe.sol:380]: `previous_amount * (actual_percent / (10 * target_percent) + 0.9);`

Comment (nuevoalex) [reply packages/augur-core/source/contracts/reporting/Universe.sol:380]: true! will change

**Pull request merged**